### ISSUE TITLE
UIWebView evaluateJS not working.

### ISSUE THREAD

**Issue by anjandaffo** [cocos2d/cocos2d-x#16087]
In UIWebView I am loading html. html is loading properly.

I need to add class on element run time by finding element by class name.

we are trying below code but it not working.

std::string jsCode = "document.getElementsByClassName('test').forEach(function(element){element.classList.add('helloClass')});"

webView->evaluateJS(jsCode);

 but it not find any element.
even if I debug for only document it return "".

**Comment by ricardoquesada** [comment]
please, add more context like:
- what cocos2d-x version you are using
- and whether or not you are using our cocos2d html5 implementation or our JS bindigns

**Comment by anjandaffo** [comment]
@ricardoquesada  I am using cocos2d-x V3.11. I am not using cocos2d html5. I using cocos2d-x cpp and UIWebView class.

**Comment by ricardoquesada** [comment]
@anjandaffo thanks.
I could be wrong, but as far as I know our JavaScript bindings don't support DOM functionality. Only JavaScript is supported.

@minggo am I right?

**Comment by anjandaffo** [comment]
@ricardoquesada  Can you suggest any alternate solution. I need to add class on div runtime.

**Comment by ricardoquesada** [comment]
@anjandaffo  yeah... I don't think we support `<div>` and other HTML / DOM / CSS components. As far as I know we support JavaScript and certain features like WebSockets that we manually added.

But I'm not 100% sure about what we manually added, and what not.

@minggo who is our JavaScript guy? Who knows exactly what "HTML" features we implemented in JSB?

**Comment by minggo** [comment]
@ricardoquesada UI provide UIWebView widget. @zilongshanren takes charge of UI system. @zilongshanren please take a look. Thanks.

**Comment by krishanfrog** [comment]
Hello,
any update on this? is DOM supported?
